src/index.js

```diff
@@ -0,0 +1,53 @@
+import './style.css';
+import dragDropIcon from './images/drag-drop-icon.svg';
+
+const toDoTask = [
+  {
+    description: 'task1',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'task2',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'task3',
+    completed: false,
+    index: 2,
+  },
+];
+
+function getTaskList(task) {
+  const toDoTaskItem = document.createElement('li');
+  toDoTaskItem.classList.add('todo-list-item');
+  {
+    const toDoTaskStatus = document.createElement('input');
+    toDoTaskStatus.type = 'checkbox';
+    toDoTaskStatus.checked = task.completed;
+    toDoTaskStatus.classList.add('task-status');
+    toDoTaskItem.appendChild(toDoTaskStatus);
+  }
+  {
+    const toDoTaskDescription = document.createElement('p');
+    toDoTaskDescription.textContent = task.description;
+    toDoTaskDescription.classList.add('task-description');
+    toDoTaskItem.appendChild(toDoTaskDescription);
+  }
+  {
+    const toDoTaskplaceicon = document.createElement('img');
+    // toDoTaskplaceicon.src = '.images/drag-drop-icon.svg';
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻